fossunited/chapters/doctype/foss_event_rsvp/templates/foss_event_rsvp.html

```
@@ -2,101 +2,101 @@
+<meta property="og:title" content="RSVP is coming soon! - {{ event.event_name }}, {{ event.chapter_name | title }}" />
+{% endif %}
+{% if event.event_type in ['Meet Up'] %}
+<meta property="og:description"
+  content="RSVP forms are live for {{ event.event_name }} {{ event.event_start_date.strftime('%B') }} - {{ event.chapter_name | title }}" />
+{% else %}
+<meta property="og:description"
+  content="RSVP forms are live for {{ event.event_name }} {{ event.event_start_date.strftime('%B') }}" />
+{% endif %}
+
+<meta property="og:image" content="https://fossunited.org/files/og-image.png" />
+<meta property="" og:image:width"" content="" 1200″" />
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)